src/classes/ADDR_Validator_TEST.cls

```diff
@@ -35,39 +35,26 @@
 */
 @isTest
 public with sharing class ADDR_Validator_TEST {
```

Comment:
Given what this test class does, I think the class could be marked with `IsParallel=true`. That would aid in performance. The only caveat might be just making sure that Contacts and Accounts are created with the same Name in multiple test methods. I think you've moved most of the data creation into a helper method so that could easily just generate a random number to use in the name. Given that the class size is effectively doubling with all of the new tests, I think this would be a welcome relatively small refactor with a larger postiive result.

Reply:
Great point to make gradual changes to unit tests especially since sometimes code needs to be refactored in order to create a testable code. However, this class cannot be set to have parallel testing due to creating a user in a test method:
> System.DmlException: Insert failed. First exception on row 0; first error: APEX_DATA_ACCESS_RESTRICTION, In parallel Apex tests, DML operations aren't permitted on this type of object: User: []

I have just upvoted the following [idea](https://success.salesforce.com/ideaView?id=0873A000000lGvtQAE) to enable user creation in parallel test.